ui/app/components/clients/page/counts.ts

```diff
@@ -3,56 +3,35 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
-import Component from '@glimmer/component';
+import ActivityComponent from '../activity';
 import { service } from '@ember/service';
 import { action } from '@ember/object';
-import { fromUnixTime, getUnixTime, isSameMonth, isAfter } from 'date-fns';
+import { getUnixTime, isSameMonth, isAfter } from 'date-fns';
 import { parseAPITimestamp } from 'core/utils/date-formatters';
 import { formatDateObject } from 'core/utils/client-count-utils';
 
 import type VersionService from 'vault/services/version';
-import type ClientsActivityModel from 'vault/models/clients/activity';
-import type ClientsConfigModel from 'vault/models/clients/config';
 import type StoreService from 'vault/services/store';
 
-interface Args {
-  activity: ClientsActivityModel;
-  config: ClientsConfigModel;
-  startTimestamp: number;
-  endTimestamp: number;
-  currentTimestamp: number;
-  namespace: string;
-  authMount: string;
-  onFilterChange: CallableFunction;
-}
-
-export default class ClientsCountsPageComponent extends Component<Args> {
+export default class ClientsCountsPageComponent extends ActivityComponent {
   @service declare readonly version: VersionService;
   @service declare readonly store: StoreService;
 
-  get startDate() {
```

Comment:
It seemed sensible to reuse the getters defined in the Activity component so date getters are easily traced back to the response

Reply:
I'm hesitant about extending the `ActivityComponent` here since it was created with the intent that having activity is a requirement which is not necessarily the case in this route. If there are some getters that can be shared I think it would be better to pull those out into a utility.